lib/widget/dropdown.dart

```diff
@@ -212,10 +244,11 @@ class SelectOneState extends FormFieldWidgetState<SelectOne> {
 /// Data Object for a SelectOne's item
 class SelectOneItem {
   SelectOneItem({
-    required this.value,
-    this.label
+    required this.value, this.label, this.iconname, this.iconsize
   });
 
   final dynamic value;
   final String? label;
+  final String? iconname;
```

Comment:
we should just use IconModel (icon name, library, color, size) as it's already defined. Please see Utils.getIcon() and example widget like tab_bar.

Reply:
okay got it